testdata/MultilineString.cs

```diff
@@ -1,22 +1,21 @@
+using System.Collections.Generic;
```

Comment:
Do the changes hereunder relate to those above, in this commit? I think they ought to go in 37903bb, but if they are of a kind and I'm not seeing it, it's worth explaining in the commit message how they are related.

Reply:
No, they aren't related. I'll split them out to make it clearer.